node/pkg/telemetry/loki.go

```diff
@@ -142,13 +153,35 @@ func NewLokiCloudLogger(ctx context.Context, logger *zap.Logger, url string, pro
 		lokiLabels[level] = levLabels
 	}
 
+	// Create a buffered channel so the application does not block in the logger. The size of this
+	// channel should be big enough to handle a burst of log messages from the app.
+	localChan := make(chan api.Entry, 1000)
```

Comment:
It would be better to declare the `1000` as a constant, and maybe take the comment away here and add it to the defintion.